extension/textext/__init__.py

```diff
@@ -608,11 +602,16 @@ def parse_pdf_log(self, logfile):
     class TexTextElement(inkex.elements.Group):
         tag_name = "g"
 
-        def __init__(self, svg_filename=None):
+        def __init__(self, svg_filename=None, doc_unit_to_mm=1.0):
```

Comment:
1) `doc_unit_to_mm` is actually `1mm` in document units, should be named accordingly, e.g. `mm_in_uu` or `mm_in_doc_units`
2) Why `doc_unit_to_mm` has default value?